zinc/.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "files.autoSave": "onFocusChange",
```

Comment:
maybe use afterDelay as in other two configs?

Reply:
Good point, thanks